src/js/controller/PenSizeController.js

```diff
@@ -24,4 +25,19 @@
     var size = pskl.app.penSizeService.getPenSize();
     this.sizePicker.setSize(size);
   };
+
+  ns.PenSizeController.prototype.createChangePenSize = function (i18n) {
+    var templateValues = {
+      title: i18n.penSizeChangeTitle() + '</br>' + i18n.penSizeChangeAt14816(),
```

Comment:
I'm suprised the `<br>` tag can be used in the `title` attribute of an HTML element. I would expect only normal text. Would a newline character `\n` work as well?

Reply:
Going to leave as is